spec/models/post_spec.rb

```diff
@@ -58,12 +76,21 @@
         old_tagged_at
       }
 
-      it "should update edited_at but not tagged_at when content edited" do
+      it "should update edited_at and but not tagged_at when subject edited" do
         Timecop.freeze(time) do
-          post.content = 'newer content'
+          post.subject = 'new title'
           post.save!
+          expect(post.edited_at).to be > post.created_at
           expect(post.tagged_at).to be_the_same_time_as(old_tagged_at)
-          expect(post.edited_at).to be > old_edited_at
+        end
+      end
+
+      it "should not update when content edited" do
+        Timecop.freeze(time) do
+          post.written.content = 'newer content'
+          post.save!
```

Comment:
What's up with this saving the post? Is this meant to be intentionally different from the above "should update tagged_at but not edited_at when content edited"?

Reply:
saving hte post should save the content, but it should _not_ change either of those values. I should probably check it does save the content tho

Reply:
apparently it doesn't

Reply:
making them the same now xD